Device: cuda:0
----------------------(train)----------------------
Epoch 0/15
---------------------------------------------------
Traceback (most recent call last):
  File "train.py", line 65, in <module>
    no_wandb=args.no_wandb,
  File "/media/ssd4t/mana/GANomaly/model.py", line 51, in train
    print(img.size())
AttributeError: 'list' object has no attribute 'size'
